components/text/hunspell/Makefile

```diff
@@ -41,6 +42,13 @@ CPPFLAGS += -I/usr/include/ncurses
 CFLAGS += -D_XOPEN_SOURCE=600
 CFLAGS += -std=c99
 
+CPPFLAGS +=	-I/usr/include/ncurses
+
+# Check location of ncurses libraries
+ifeq ($(shell test -f /usr/gnu/lib/libncurses.so && echo y),y)
```

Comment:
Actually we have a problem, our current component still refers to /usr/gnu/lib:
helios> pkg-config --libs ncurses
-L/usr/gnu/lib -lncurses

I am not a big fan of this change, better patch the build systems to rely on pkg-config.